CHANGELOG.md

```diff
@@ -122,12 +122,37 @@ binary | sha1 hash | md5 hash
   * Display line number on JSON errors
   * Add flag -t as shorthand for --tty
 * Resources
+  * Improved node stability by *optionally* evicting pods upon memory pressure - [Design Doc](https://github.com/kubernetes/kubernetes/blob/release-1.3/docs/proposals/kubelet-eviction.md) 
   * **alpha**: NVIDIA GPU support ([#24836](https://github.com/kubernetes/kubernetes/pull/24836), [@therc](https://github.com/therc))
   * Adding loadBalancer services and nodeports services to quota system
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *[v1.10](https://github.com/kubernetes/kubernetes/issues/19720)*, *[v1.11](https://github.com/kubernetes/kubernetes/issues/23397)*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+All prior versions of docker will not be supported.
+
+#### kubernetes Core Known Issues
```

Comment:
The rendering was not great with `###`. I can switch it back if you think that's not an issue.